### DISCUSSION TITLE
HTTP Error 413 - Request Entity Too Large

### DISCUSSION THREAD

**Discussion by pietz** [deta/cloud-docs#566] [Help]
I'm getting this error when sending a binary file of 7MB to one of my endpoints that I want to store in Deta Drive. Since it doesn't show in the Visor logs, I believe it's a Deta Micro limitation.

How can one of my users upload a file of this size or larger in Deta Drive if Micro doesnt't accepts requests larger than (what seems to be) 5MB?

EDIT: Possibly relevant https://stackoverflow.com/questions/52599721/maximum-size-of-payload-for-an-aws-lambda-request-event - Is the described workaround possible with Deta?

**Comment by AyushSehrawat** [comment]
Deta has a payload limit of 5.5mb so you can't upload files greater than that, but @jnsougata figured a way out... Wait for his response probably!

**Reply by jnsougata** [reply]
just mimic chunk upload using micro endpoint

**Reply by pietz** [reply]
Thanky guys! @jnsougata you seem very experienced with this. Do you mind providing a code sample? Preferably for FastAPI?

I think the solution might come in handy for more people and should be documented.

**Reply by jnsougata** [reply]
I'm not aware how the method will scale... that's just a theory
